support-frontend/assets/pages/subscriptions-landing/copy/subscriptionCopy.tsx

```diff
@@ -61,19 +61,26 @@ const getDisplayPrice = (
 	return `${currency}${fixDecimals(price)}/${billingPeriod}`;
 };
 
-const getDigitalEditionPrices = (countryGroupId: CountryGroupId): string => {
+const getDigitalEditionPrice = (
+	countryGroupId: CountryGroupId,
+	billingPeriod: BillingPeriod,
+): string => {
 	const currencyKey = detect(countryGroupId);
 	const currency = currencies[currencyKey].glyph;
 	const product = productCatalog['DigitalSubscription'];
-	const priceMonthly = product?.ratePlans[Monthly]?.pricing[currencyKey];
-	const currencyPriceMonthly = priceMonthly
-		? `${currency}${fixDecimals(priceMonthly)}/${Monthly}`
+	const price = product?.ratePlans[billingPeriod]?.pricing[currencyKey];
+	return price
+		? `${currency}${fixDecimals(price)}/${
+				billingPeriod === Annual ? 'Annually' : billingPeriod
+		  }`
 		: '';
-	const priceAnnual = product?.ratePlans[Annual]?.pricing[currencyKey];
-	const currencyPriceAnnual = priceAnnual
-		? ` ${currency}${fixDecimals(priceAnnual)}/${Annual}`
-		: '';
-	return `${currencyPriceMonthly}${currencyPriceAnnual}`;
+};
+const getDigitalEditionPrices = (countryGroupId: CountryGroupId): string => {
+	const priceMonthly = getDigitalEditionPrice(countryGroupId, Monthly);
+	return `${priceMonthly}${priceMonthly ? ' or' : ''}${getDigitalEditionPrice(
+		countryGroupId,
+		Annual,
+	)}`;
```

Comment:
I would avoid having ternary inside string interpolations as it's very difficult to read. Also, is priceMonthly ever undefined? if not we could go with:
```suggestion
	const priceMonthly = getDigitalEditionPrice(countryGroupId, Monthly);
	const priceAnnual = getDigitalEditionPrice(countryGroupId, Annual);
	return `${priceMonthly} or ${priceAnnual}`;
```

Reply:
@maloky : I think its 'possible' there could be only a monthly or yearly price (perhaps the 'or' part in the middle is probably needed?). Agree looks more readable with `getDigitalEditionPrice(countryGroupId, Annual)` so put that through for readability.  Hope thats ok, updated.